examples/starwars/service.bal

```diff
@@ -58,10 +70,12 @@ service /graphql on new graphql:Listener(9000) {
     resource function get characters(string[] idList) returns Character?[] {
         Character[] characters = [];
         foreach string id in idList {
-            if ds:humanTable[id] is ds:HumanRecord {
-                characters.push(new Human(<ds:HumanRecord> ds:humanTable[id]));
-            } else if ds:droidTable[id] is ds:DroidRecord {
-                characters.push(new Droid(<ds:DroidRecord> ds:droidTable[id]));
+            if ds:humanTable.hasKey(id) {
+                characters.push(new Human(ds:humanTable.get(id)));
+            } else if ds:droidTable.hasKey(id) {
+                characters.push(new Droid(ds:droidTable.get(id)));
+            } else {
+                characters.push();
```

Comment:
don't we have to push nil here ?

Reply:
Fixed with 1d4f453047b0cfb1db1e0570a17cf4617994110f